dashboard/modules/serve/sdk.py

```diff
@@ -11,10 +11,10 @@
 from ray.dashboard.modules.dashboard_sdk import SubmissionClient
 
 
-DEPLOY_PATH = "/api/serve/deployments/"
-INFO_PATH = "/api/serve/deployments/"
+DEPLOY_PATH = "/api/serve/deployments"
+INFO_PATH = "/api/serve/deployments"
 STATUS_PATH = "/api/serve/deployments/status"
-DELETE_PATH = "/api/serve/deployments/"
+DELETE_PATH = "/api/serve/deployments"
 
```

Comment:
should we have the corresponding application paths?

Reply:
I'll make the corresponding changes in `sdk.py` when update the CLI in a follow up PR!